shared/js/background/message-handlers.js

```diff
@@ -361,13 +361,26 @@ export function getListContents (list) {
     }
 }
 
-export function setListContents ({ name, value }) {
+/**
+ * Manually override the value of a list
+ * @param {{ name: string, value: object}} list value
+ */
+export async function setListContents ({ name, value }) {
```

Comment:
Maybe we should give this a comment or name that makes it clear it's only for testing purposes? E.g. `setListContentsForTest` or something?

Reply:
This is also used by the `list-editor` debug page.